src/crustls.h

```diff
@@ -304,6 +304,21 @@ typedef enum rustls_result (*rustls_session_store_get_callback)(rustls_session_s
  */
 typedef enum rustls_result (*rustls_session_store_put_callback)(rustls_session_store_userdata userdata, const struct rustls_slice_bytes *key, const struct rustls_slice_bytes *val);
 
+/**
+ * Any context information the callback will receive when invoked.
+ */
+typedef void *rustls_ocsp_userdata;
+
+/**
+ * Prototype of a callback that retrieves OCSP response data (DER format)
+ * for the given `certified_key`. The OCSP data has to be copied into the
+ * provided `buf`, if it's length is sufficient. The number of copied bytes
+ * need to be returned in `out_n`.
+ * If `buf` is not of sufficient size, or if not OCSP data is available,
+ * `out_n` must be set to 0.
```

Comment:
```suggestion
 * Prototype of a callback that retrieves OCSP response data (DER format)
 * for the given `certified_key`. The OCSP data has to be copied into the
 * provided `buf`, if its length is sufficient. The number of copied bytes
 * need to be returned in `out_n`.
 * If `buf` is not of sufficient size, or if no OCSP data is available,
 * `out_n` must be set to 0.
```
What happens if the client sets `out_n` to 0? Does the function that the callback was passed to return an error?